src/main/java/com/javadeobfuscator/deobfuscator/transformers/normalizer/VariableNormalizer.java

```diff
@@ -1,40 +0,0 @@
-/*
- * Copyright 2016 Sam Sun <me@samczsun.com>
- *
- *    Licensed under the Apache License, Version 2.0 (the "License");
- *    you may not use this file except in compliance with the License.
- *    You may obtain a copy of the License at
- *
- *        http://www.apache.org/licenses/LICENSE-2.0
- *
- *    Unless required by applicable law or agreed to in writing, software
- *    distributed under the License is distributed on an "AS IS" BASIS,
- *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- *    See the License for the specific language governing permissions and
- *    limitations under the License.
- */
-
-package com.javadeobfuscator.deobfuscator.transformers.normalizer;
-
-import com.javadeobfuscator.deobfuscator.config.TransformerConfig;
-import com.javadeobfuscator.deobfuscator.transformers.Transformer;
-
-import java.util.concurrent.atomic.AtomicInteger;
-
-public class VariableNormalizer extends Transformer<TransformerConfig> {
```

Comment:
Why is this removed?

Reply:
This was removed because it was a bad solution to the illegal local variables problem. Removing them entirely worked better for decompilers.